server/src/utils/models/url.js

```diff
@@ -0,0 +1,27 @@
+const Sequelize = require("sequelize");
+
+const { sequelizeConnection } = require("../../databases/postgres/database");
+
+const sequelizeUrlDB = sequelizeConnection();
+
+const getUrlModel = () => {
+  const tableName =
```

Comment:
we want to call a generic table name from the env vars. It will benefit us in the future when we want to add more deployment envs. Future proofing it.